wake-lock/wakelock-api.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Wake Lock API Test</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
```

Comment:
There will probably be quite a few people authoring this and will likely confuse people. Please add a C0 or W3C license to this instead.